server/index.js

```diff
@@ -146,6 +146,26 @@ app.post('/api/cart', (req, res, next) => {
     .catch(err => next(err));
 });
 
+app.post('/api/orders', (req, res, next) => {
+  if (!sessionCartId) {
+    next(new ClientError('There doesn\'t seem to be a cart available for checkout', 400));
+  } else if (!req.body.name || !req.body.creditCard || !req.body.shippingAddress) {
+    next(new ClientError('A name, credit card number, and shipping address must be provided', 400));
```

Comment:
Be sure to `return` here or else your back end may try to respond twice and crash